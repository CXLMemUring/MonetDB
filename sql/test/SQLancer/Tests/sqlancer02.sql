START TRANSACTION; -- Bug 6909
CREATE TABLE "sys"."t0" ("c0" BOOLEAN,"c1" DOUBLE PRECISION NOT NULL);
COPY 4 RECORDS INTO "sys"."t0" FROM stdin USING DELIMITERS E'\t',E'\n','"';
false	0.2059926383949241
NULL	1.89755202e+09
NULL	0.8514002985569765
NULL	0.8565135463082767

SELECT ALL t0.c0 FROM t0 WHERE (((0.654013919354451) BETWEEN SYMMETRIC (t0.c1) AND ("length"(upper(''))))OR(t0.c0));
	-- NULL
	-- NULL
	-- NULL
SELECT t0.c0 FROM t0 WHERE 1 BETWEEN SYMMETRIC t0.c1 AND 0;
	-- NULL
SELECT t0.c0 FROM t0 WHERE 1 BETWEEN t0.c1 AND 0;
	-- empty
SELECT ALL t0.c0 FROM t0 WHERE (((0.654013919354451) BETWEEN SYMMETRIC (t0.c1) AND ("length"(upper(''))))OR(t0.c0)) UNION ALL 
SELECT ALL t0.c0 FROM t0 WHERE NOT ((((0.654013919354451) BETWEEN SYMMETRIC (t0.c1) AND ("length"(upper(''))))OR(t0.c0))) UNION ALL 
SELECT t0.c0 FROM t0 WHERE ((((0.654013919354451) BETWEEN SYMMETRIC (t0.c1) AND ("length"(upper(''))))OR(t0.c0))) IS NULL;
	-- NULL
	-- NULL
	-- NULL
	-- False
ROLLBACK;

START TRANSACTION; -- Bug 6910
CREATE TABLE t0("c0" DOUBLE NOT NULL);
COPY 11 RECORDS INTO "sys"."t0" FROM stdin USING DELIMITERS E'\t',E'\n','"';
0
0.9440456550171384
0.3985168253176739
0.9033732171086648
0.969477015070214
-4798112
0.6321209051017095
0.7740272412444077
0.7778437505533593
0.34293653568934335
0.1228907839970418

SELECT AVG(ALL ((((CAST(((+ (-266455613))/(sql_min(-384923592, -684635832))) AS INT))+(abs(((1380859174)*(1039801791))))))*(((((greatest(1447838997, -1100245247))/(((((849535137)||(-1532296761)))/(CAST(t0.c0 AS INT))))))*(- (+ (((-291947972)*(626914646))))))))), 0.77785585904529674916574322196538560092449188232421875 FROM t0 WHERE (0.06276420221300283) NOT  BETWEEN ASYMMETRIC (t0.c0) AND (t0.c0) 
GROUP BY 0.135845384683200176567652306403033435344696044921875, 0.86688447, 0.3299475775341662586725988148828037083148956298828125, + (+ (- (length(r'q')))), greatest(upper(upper(r'623865997')), trim(replace(r'vAd(Lup}Z', r'-1774953973', r'0.4235712099498189'), substr(r'', -1963236744)));
ROLLBACK;
