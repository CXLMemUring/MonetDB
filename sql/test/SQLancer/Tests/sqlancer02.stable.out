stdout of test 'sqlancer02` in directory 'sql/test/SQLancer` itself:


# 17:04:12 >  
# 17:04:12 >  "mclient" "-lsql" "-ftest" "-tnone" "-Eutf-8" "-i" "-e" "--host=/var/tmp/mtest-438766" "--port=32798"
# 17:04:12 >  

#START TRANSACTION; -- Bug 6909
#CREATE TABLE "sys"."t0" ("c0" BOOLEAN,"c1" DOUBLE PRECISION NOT NULL);
#COPY 4 RECORDS INTO "sys"."t0" FROM stdin USING DELIMITERS E'\t',E'\n','"';
#false	0.2059926383949241
#NULL	1.89755202e+09
#NULL	0.8514002985569765
#NULL	0.8565135463082767
[ 4	]
#SELECT ALL t0.c0 FROM t0 WHERE (((0.654013919354451) BETWEEN SYMMETRIC (t0.c1) AND ("length"(upper(''))))OR(t0.c0));
% sys.t0 # table_name
% c0 # name
% boolean # type
% 5 # length
[ NULL	]
[ NULL	]
[ NULL	]
#SELECT t0.c0 FROM t0 WHERE 1 BETWEEN SYMMETRIC t0.c1 AND 0;
% sys.t0 # table_name
% c0 # name
% boolean # type
% 5 # length
[ NULL	]
#SELECT t0.c0 FROM t0 WHERE 1 BETWEEN t0.c1 AND 0;
% sys.t0 # table_name
% c0 # name
% boolean # type
% 5 # length
#SELECT ALL t0.c0 FROM t0 WHERE (((0.654013919354451) BETWEEN SYMMETRIC (t0.c1) AND ("length"(upper(''))))OR(t0.c0)) UNION ALL 
#SELECT ALL t0.c0 FROM t0 WHERE NOT ((((0.654013919354451) BETWEEN SYMMETRIC (t0.c1) AND ("length"(upper(''))))OR(t0.c0))) UNION ALL 
#SELECT t0.c0 FROM t0 WHERE ((((0.654013919354451) BETWEEN SYMMETRIC (t0.c1) AND ("length"(upper(''))))OR(t0.c0))) IS NULL;
% .%12 # table_name
% c0 # name
% boolean # type
% 5 # length
[ NULL	]
[ NULL	]
[ NULL	]
[ false	]
#ROLLBACK;
#START TRANSACTION;
#CREATE TABLE integers(i INTEGER);
#INSERT INTO integers VALUES (1), (2), (3), (NULL);
[ 4	]
#select i between symmetric cast(1 as decimal) and cast(2 as double) from integers;
% sys.%3 # table_name
% %3 # name
% boolean # type
% 5 # length
[ true	]
[ true	]
[ false	]
[ NULL	]
#ROLLBACK;
#START TRANSACTION; -- Bug 6910
#CREATE TABLE t0("c0" DOUBLE NOT NULL);
#COPY 11 RECORDS INTO "sys"."t0" FROM stdin USING DELIMITERS E'\t',E'\n','"';
#0
#0.9440456550171384
#0.3985168253176739
#0.9033732171086648
#0.969477015070214
#-4798112
#0.6321209051017095
#0.7740272412444077
#0.7778437505533593
#0.34293653568934335
#0.1228907839970418
[ 11	]
#ROLLBACK;

# 17:04:12 >  
# 17:04:12 >  "Done."
# 17:04:12 >  

